code/datums/traits/_quirk.dm

```diff
@@ -10,6 +10,7 @@
 	var/medical_record_text //This text will appear on medical records for the trait. Not yet implemented
 	var/mood_quirk = FALSE //if true, this quirk affects mood and is unavailable if moodlets are disabled
 	var/mob_trait //if applicable, apply and remove this mob trait
+	var/process = FALSE // Does this proc use on_process()?
 	var/mob/living/quirk_holder
 
 /datum/quirk/New(mob/living/quirk_mob, spawn_effects)
```

Comment:
Might be worth putting a `SHOULD_CALL_PARENT(TRUE)` in here considering how important it is.

Reply:
should_call_parent is only really used on proc definitions. I can add it to this if you want but it seems pretty moot given the thousands of much more important parent procs that don't use it 😆